apps/state/lib/state/connecting_stops.ex

```diff
@@ -186,6 +186,6 @@ defmodule State.ConnectingStops do
 
   @spec patterns_at_stop(Stop.t()) :: MapSet.t(RoutePattern.id())
   defp patterns_at_stop(%{id: id}) do
-    [id] |> RoutesPatternsAtStop.route_patterns_by_family_stops(ignore?: false) |> MapSet.new()
+    [id] |> RoutesPatternsAtStop.route_patterns_by_family_stops(canonical?: false) |> MapSet.new()
```

Comment:
I haven't gotten to this code change yet, so I might answer my question later: but should this be `canonical?: true`, since before it was `ignore?: false`?

Reply:
They mean the same thing. `ignore?: false` means "don't ignore", i.e. include trips that are `ignored` — `canonical?: false` means "non-canonical", i.e. include trips that are not `canonical`. Since `ignored == !canonical`, the value should be unchanged.